Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -152,6 +152,13 @@ def getResultSummary(self):
         else:
             return {'step': 'Cleaned and updated working directory'}
 
+    @defer.inlineCallbacks
+    def run(self):
+        rc = yield super().run()
+        if rc == SUCCESS:
+            yield self._dovccmd(['remote', 'set-url', '--push', 'origin', 'INVALID'])
```

Comment:
maybe change INVALID to something like PUSH_DISABLED_BY_ADMIN. So it it's easy to debug when someone sees the error message (basically error message will specify why origin is invalid).